### ISSUE TITLE
Bug impossible to get a username

### ISSUE THREAD

**Issue by You2s** [taylornetwork/laravel-username-generator#23]
First of all, thank you for the very fast addition of the feature, it's the first time I've seen such a reactivity!
Unfortunately I have a problem, I installed the package, published the configuration (php artisan vendor:publish), and tried to use the package with tinker (php artisan tinker) but it doesn't seem to work:
![Capture d’écran 2019-10-31 à 20 17 15](https://user-images.githubusercontent.com/7408320/67978912-7c39f380-fc1b-11e9-8a11-0f1c04a0aa78.png)

![Capture d’écran 2019-10-31 à 20 18 23](https://user-images.githubusercontent.com/7408320/67978959-9ecc0c80-fc1b-11e9-9425-adc1fa5c56cd.png)

Can you help me ?

**Comment by samueljtaylor** [comment]
Hi @You2s,

That's very strange, I've run the tests and they all pass and then I spun it up in a new app and I'm not getting that issue.

What version of Laravel are you using?

**Comment by You2s** [comment]
I'm using laravel 5.7

**Comment by samueljtaylor** [comment]
Can you run `composer dump-autoload` and then under tinker 

```
>>>  UsernameGenerator::generate();
>>>  UsernameGenerator::generate('Test User');
```

And see if that fixes it?

**Comment by You2s** [comment]
Already tried, same result

**Comment by You2s** [comment]
I tried this to add a debug in the makeUnique function of the BaseDriver.php file :
```
/**
     * Make the username unique.
     *
     * @param string $text
     *
     * @return string
     */
    public function makeUnique(string $text): string
    {
        if ($this->getConfig('unique') && $this->model() && method_exists($this->model(), 'findSimilarUsernames')) {
            Log::debug($this->model()->findSimilarUsernames($text));
            if (($similar = count($this->model()->findSimilarUsernames($text))) > 0) {
                return $text.$this->getConfig('separator').$similar;
            }
        }

        return $text;
    }
```
and get this:
`[2019-10-31 20:02:00] local.DEBUG: 0`

**Comment by You2s** [comment]
I may have found, in the findSimilarUsernames function, there is a return 0; if the username does not exist in db, this returned 0 is used as a parameter in the count() function that expects an object. 
I tried to replace this:
```
public function findSimilarUsernames($username)
    {
        if (count(static::where(config('username_generator.column', 'username'), $username)->get()) === 0) {
            return 0;
        }

        return static::where(config('username_generator.column', 'username'), 'LIKE', $username.'%')->get();
    }
```
By this: 
```
public function findSimilarUsernames($username)
    {
        if (count(static::where(config('username_generator.column', 'username'), $username)->get()) === 0) {
            return static::where(config('username_generator.column', 'username'), $username)->get();
        }

        return static::where(config('username_generator.column', 'username'), 'LIKE', $username.'%')->get();
    }
```
And it's works

**Comment by samueljtaylor** [comment]
You're 100% right, I'll fix right now.

**Comment by You2s** [comment]
Thank you for your reactivity

**Comment by samueljtaylor** [comment]
No problem! I've published release v2.3.1  that fixes the issue. Appreciate the help in locating the bug!
